nestkernel/spike_data.h

```diff
@@ -128,9 +136,13 @@ class SpikeData
   double get_offset() const;
 };
 
+//!< check legal size
+typedef StaticAssert< sizeof( SpikeData ) == 8 >::success
+  success_spike_data_size;
```

Comment:
Since we use C++11 now, maybe use type alias instead? So
```c++
typedef StaticAssert< sizeof( SpikeData ) == 8 >::success success_spike_data_size;
```
would become
```c++
using success_spike_data_size = StaticAssert< sizeof( SpikeData ) == 8 >::success;
```

Reply:
is this best practice now?